hexrd/ui/hand_drawn_mask_dialog.py

```diff
@@ -157,6 +168,13 @@ def rejected(self):
     def show(self):
         self.ui.show()
 
+    def tabbed_view_changed(self):
+        if self.linebuilder:
+            self.line_finished()
+
+    def tabbed_mode_changed(self):
+        self.accepted()
```

Comment:
```suggestion
        self.accept()
```

Reply:
I don't understand this suggestion? We want to call `accepted` to complete the current line and there is no `accept` function.

Reply:
Oh I see, sorry. I was thinking about `QDialog`s where `accepted()` is a signal but `accept()` is a slot, and we would typically prefer calling the slot.

No need to change anything here.